package.json

```diff
@@ -323,6 +324,41 @@
                         }
                       }
                     },
+                    {
+                      "if": {
+                        "properties": {
+                          "connectionType": {
+                            "const": "iom"
+                          }
+                        }
+                      },
+                      "then": {
+                        "required": [
+                          "host",
+                          "username",
+                          "port"
+                        ],
+                        "properties": {
+                          "host": {
+                            "type": "string",
+                            "default": "",
+                            "description": "%configuration.SAS.connectionProfiles.profiles.ssh.host%"
+                          },
+                          "username": {
+                            "type": "string",
+                            "default": "",
+                            "description": "%configuration.SAS.connectionProfiles.profiles.ssh.username%"
+                          },
+                          "port": {
+                            "type": "number",
+                            "default": 22,
```

Comment:
8591?

Reply:
Resolved in https://github.com/sassoftware/vscode-sas-extension/pull/592/commits/1d7ee717f30707e321a05af5152a4c8ea2b388b9